justfile

```diff
@@ -34,6 +34,10 @@ cpp-format:
     fd --extension c --exec clang-format -i
     fd --extension cpp --exec clang-format -i
 
+# Build our C++ SDK, and all our tests and examples
+cpp-build-all:
+    pixi run cpp-build-all
```

Comment:
uhoh. I think I put this on some other pr as well, or did I discard it again? We'll notice 😄

Reply:
Sadly it doesn't build `tests/cpp/roundtrips/tensor/main.cpp`